main/templates/index.html

```diff
@@ -2,9 +2,20 @@
 
 {% block titulo %} Página inicial {% endblock %}
 
+{% block extrahead %} 
+{% load static %}
+<link rel="stylesheet" type="text/css" href="{% static 'styles/index.css' %}"> 
+<script src="//code.jquery.com/jquery-1.11.1.min.js"></script>
```

Comment:
jquery is not being used in this template i think

Reply:
correct, i'm going to change that